src/App.tsx

```diff
@@ -1,35 +1,29 @@
 import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import Counter from './components/counter';
+import Header from './components/header/header';
+import './App.css';
 
 function App() {
-  const [count, setCount] = useState(0)
+  let genreList=["Horror","Romantic","Thrilling", "Comedy"];
+  const [selectedGenre, setSelectedGenre] = useState('Horror');
 
+  const onSelectGenre = (genre:string) => () => {
+    console.log(genre);
+    setSelectedGenre(genre);
+}
   return (
     <>
-      <div>
-        <a href="https://vitejs.dev" target="_blank">
-          <img src={viteLogo} className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://react.dev" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>Vite + React</h1>
-      <div className="card">
-        <button onClick={() => setCount((count) => count + 1)}>
-          count is {count}
-        </button>
-        <p>
-          Edit <code>src/App.tsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
+        <Counter initialValue={1} /> 
```

Comment:
The counter is not visible in the viewport with this composition.

Reply:
It was existing however hidden , I had to put it under the <div> layout.